src/plots/cartesian/axes.js

```diff
@@ -34,6 +34,14 @@ var ONESEC = constants.ONESEC;
 var MINUS_SIGN = constants.MINUS_SIGN;
 var BADNUM = constants.BADNUM;
 
+var VISIBLE = { display: null };
+var HIDDEN = { display: 'none' };
```

Comment:
Per https://github.com/plotly/plotly.js/pull/5550#discussion_r594815159 we should not add new instances of the single-arg form of `selection.style`, these should all be `.style('display', null)` etc.

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/5550/commits/f05cd39ca199b0395ef340998cf5e977a1676f06.